### ISSUE TITLE
Elements and Attributes have id="undefined" in Working Draft

### ISSUE THREAD

**Issue by carmacleod** [w3c/html-aam#255]
The _Editor's Draft_ version of the HTML-AAM spec has valid id's that provide a link to each element in the [HTML Element Role Mappings Table](https://w3c.github.io/html-aam/#html-element-role-mappings).

For example, the anchor element id is `el-a`, so its link is: https://w3c.github.io/html-aam/#el-a

The code in the Editor's Draft is:
```html
<summary id="el-a" role="button" aria-expanded="true" aria-controls="details-id-0">
```

However, in the _Working Draft_ version, this id is `undefined`.

To link to the anchor element in the Working Draft, one must use the difficult-to-discover `details-id-0` in the summary, which does not feel very permanent: https://www.w3.org/TR/html-aam/#details-id-0

The broken line of code is:
```html
<summary id="undefined" role="button" aria-expanded="false" aria-controls="details-id-0">
```
---
Same problem with the [HTML Attribute State and Property Mappings Table](https://w3c.github.io/html-aam/#html-attribute-state-and-property-mappings).
The working code in the Editor's Draft is:
```html
<summary id="att-abbr" role="button" aria-expanded="false" aria-controls="details-id-141">
```
And the broken code in the Working Draft is:
```html
<summary id="undefined" role="button" aria-expanded="false" aria-controls="details-id-141">
```
---
Same problem with the "Single Table View" for both tables, except that in the Working Draft version of the Table Views, there's no id at all (not even undefined).

@marcoscaceres Are these ReSpec issues?

**Comment by marcoscaceres** [comment]
I see it's unrelated to ReSpec (kinda), but thanks for letting me know... if there is something we should be doing in ReSpec core, please let me know.

**Comment by carmacleod** [comment]
> if there is something we should be doing in ReSpec core, please let me know.

Thank-you, @marcoscaceres!

@ZoeBijl Letting you know that @marcoscaceres has kindly offered help with https://github.com/w3c/aria-common/issues/7 on the ReSpec side if needed.

**Comment by ZoeBijl** [comment]
Excellent! I’ll reach out when I start work on it 👍.

**Comment by scottaohara** [comment]
related to #360.
https://github.com/w3c/core-aam/pull/105 may finally fix this issue

**Comment by scottaohara** [comment]
This is closed via merging #406
